README.md

```diff
@@ -9,16 +9,10 @@ Currently the path to the compilers, ramdisks and kernels are hardcoded, but eas
 
 ## Usage:
 
-1. Clone directly into your AOSP tree (or make sure `ANDROID_BUILD_TOP` is set to the right path).
-2. Call `./oot/oot.sh <device name>`
-   Keep in mind that not all devices have a script yet.
+1. Clone directly into your AOSP tree (or make sure `ANDROID_BUILD_TOP` is set to the right path);
```

Comment:
Maybe mention that `ANDROID_BUILD_TOP` is a variable with `${ANDROID_BUILD_TOP}`?

Reply:
Done :)